examples/gno.land/p/demo/grc/grc721/igrc721.gno

```diff
@@ -3,13 +3,13 @@ package grc721
 import "std"
 
 type IGRC721 interface {
-	BalanceOf(owner std.Address) (count int64)
-	OwnerOf(tid TokenID) std.Address
-	SafeTransferFrom(from, to std.Address, tid TokenID)
-	TransferFrom(from, to std.Address, tid TokenID)
-	Approve(approved std.Address, tid TokenID)
-	SetApprovalForAll(operator std.Address, approved bool)
-	GetApproved(tid TokenID) std.Address
+	BalanceOf(owner std.Address) (uint64, error)
+	OwnerOf(tid TokenID) (std.Address, error)
+	SafeTransferFrom(from, to std.Address, tid TokenID) error
+	TransferFrom(from, to std.Address, tid TokenID) error
+	Approve(approved std.Address, tid TokenID) error
+	SetApprovalForAll(operator std.Address, approved bool) error
+	GetApproved(tid TokenID) (std.Address, error)
 	IsApprovedForAll(owner, operator std.Address) bool
 }
```

Comment:
Please see my previous comment about what GRC721 should be modeled after.

Additionally, if you choose to keep `BalanceOf`, what do you think about making it return a simple `0` value, instead of an error?

Reply:
I think it's okay to return an error here. Later realm implementors can decide whether they want to handle the error or simply want to return 0.